make_pattern:
  board_size: [1, 1]
  tile_size: 0.05
  tile_offset: 0.0